docs/doc_4_authentication.md

```diff
@@ -95,4 +95,11 @@ revokeConfig.RevokeToken(context.Background(), &auth.Token{
 	AccessToken: "yourTokenHere"
 });
 ```
+### OAuth Token Cache
+
+Service Account OAuth Access Tokens are valid for an extended duration of 1 hour. 
```

Comment:
Just in case this number changes
```suggestion
Service Account OAuth Access Tokens validity is expressed as a number of seconds in the  `expires_in` field. 
```